.clippy.toml

```diff
@@ -0,0 +1 @@
+assigning_clones = false
```

Comment:
Missing newline at the end of the file.

Reply:
Yeah that doesn't work (that's why CI is failing). I don't think we can disable lints at the workspace level; I'll have to add it to each crate.